### PR TITLE
Moved api server folder from dataflow-ux to numaflow under numaproj

### PR DESCRIPTION
**Pull request by Krithika3**
Signed-off-by: Krithika Vijayakumar <krithika_vijayakumar@intuit.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -25,20 +27,21 @@ require (
...
-	google.golang.org/protobuf v1.27.1
-	k8s.io/api v0.23.3
-	k8s.io/apimachinery v0.23.3
-	k8s.io/client-go v0.23.3
-	k8s.io/code-generator v0.23.3
-	k8s.io/gengo v0.0.0-20210813121822-485abfe95c7c
-	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65
-	k8s.io/utils v0.0.0-20211116205334-6203023598ed
+	google.golang.org/protobuf v1.28.0
+	k8s.io/api v0.24.2
```
Comment (Krithika3) [inline go.mod:37]: Are these updates acceptable?

Comment (whynowy) [reply go.mod:37]: should be good.

Comment (whynowy) [reply go.mod:37]: hold on.... the versions are not consistent... let's use `v0.23.3`.

Comment (Krithika3) [reply go.mod:37]: I am going to close this PR and start from scratch

**Pull request closed without merging**